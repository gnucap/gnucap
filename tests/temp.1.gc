spice
.MODEL TESTR RES (TC1=1m TC2=0 DEFW=1 RSH=1)
.options temperature=37 list
R1 1 0 TESTR(w=2, l=4)
.print tran r(R1) temp(0)
.print ac r(R1) temp(0)
.print op r(R1) temp(0)
.print dc r(R1) temp(0)
.ac 1
.dc
.dc temp=400
.ac 1
.tran 0 1 .5 temp=1342
.ac 1
.dc
.ac 1
.tran 2
.ac 1
.dc 1000
.ac 1
.dc 0 1000 100
.ac 1 temp=10000
.ac 1
.dc
.ac 1
.op
.op temp=400
.ac 1
.tran 0 1 .5 temp=1342
.ac 1
.op
.ac 1
.tran 2
.ac 1
.op 1000
.ac 1
.op 0 1000 100
.ac 1 temp=10000
.ac 1
.op
.ac 1
.status notime
.end
