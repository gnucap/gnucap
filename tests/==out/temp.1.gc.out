"R1 1 0 TESTR(w=2, l=4) "
".print tran r(R1) temp(0) "
".print ac r(R1) temp(0) "
".print op r(R1) temp(0) "
".print dc r(R1) temp(0) "
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         1.946      0.        
".dc "
#           r(R1)      temp(0)   
 0.         2.02       37.       
".dc temp=400 "
#           r(R1)      temp(0)   
 0.         2.746      400.      
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         2.746      400.      
".tran 0 1 .5 temp=1342 "
#Time       r(R1)      temp(0)   
 0.         4.63       1.342K    
 0.5        4.63       1.342K    
 1.         4.63       1.342K    
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         4.63       1.342K    
".dc "
#           r(R1)      temp(0)   
 0.         2.02       37.       
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         2.02       37.       
".tran 2 "
#Time       r(R1)      temp(0)   
 0.         2.02       37.       
 0.5        2.02       37.       
 1.         2.02       37.       
 1.5        2.02       37.       
 2.         2.02       37.       
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         2.02       37.       
".dc 1000 "
#           r(R1)      temp(0)   
 1.K        2.02       37.       
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         2.02       37.       
".dc 0 1000 100 "
#           r(R1)      temp(0)   
 0.         2.02       37.       
 100.       2.02       37.       
 200.       2.02       37.       
 300.       2.02       37.       
 400.       2.02       37.       
 500.       2.02       37.       
 600.       2.02       37.       
 700.       2.02       37.       
 800.       2.02       37.       
 900.       2.02       37.       
 1.K        2.02       37.       
".ac 1 temp=10000 "
#Freq       r(R1)      temp(0)   
 1.         21.946     10.K      
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         21.946     10.K      
".dc "
#           r(R1)      temp(0)   
 0.         2.02       37.       
 100.       2.02       37.       
 200.       2.02       37.       
 300.       2.02       37.       
 400.       2.02       37.       
 500.       2.02       37.       
 600.       2.02       37.       
 700.       2.02       37.       
 800.       2.02       37.       
 900.       2.02       37.       
 1.K        2.02       37.       
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         2.02       37.       
".op "
#           r(R1)      temp(0)   
 37.        2.02       37.       
".op temp=400 "
#           r(R1)      temp(0)   
 37.        2.02       37.       
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         2.02       37.       
".tran 0 1 .5 temp=1342 "
#Time       r(R1)      temp(0)   
 0.         4.63       1.342K    
 0.5        4.63       1.342K    
 1.         4.63       1.342K    
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         4.63       1.342K    
".op "
#           r(R1)      temp(0)   
 37.        2.02       37.       
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         2.02       37.       
".tran 2 "
#Time       r(R1)      temp(0)   
 0.         2.02       37.       
 0.5        2.02       37.       
 1.         2.02       37.       
 1.5        2.02       37.       
 2.         2.02       37.       
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         2.02       37.       
".op 1000 "
#           r(R1)      temp(0)   
 1.K        3.946      1.K       
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         3.946      1.K       
".op 0 1000 100 "
#           r(R1)      temp(0)   
 0.         1.946      0.        
 100.       2.146      100.      
 200.       2.346      200.      
 300.       2.546      300.      
 400.       2.746      400.      
 500.       2.946      500.      
 600.       3.146      600.      
 700.       3.346      700.      
 800.       3.546      800.      
 900.       3.746      900.      
 1.K        3.946      1.K       
".ac 1 temp=10000 "
#Freq       r(R1)      temp(0)   
 1.         21.946     10.K      
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         21.946     10.K      
".op "
#           r(R1)      temp(0)   
 0.         1.946      0.        
 100.       2.146      100.      
 200.       2.346      200.      
 300.       2.546      300.      
 400.       2.746      400.      
 500.       2.946      500.      
 600.       3.146      600.      
 700.       3.346      700.      
 800.       3.546      800.      
 900.       3.746      900.      
 1.K        3.946      1.K       
".ac 1 "
#Freq       r(R1)      temp(0)   
 1.         3.946      1.K       
".status notime "
Gnucap   System status
iterations: op=33, dc=23, tran=13, fourier=0, total=217
transient timesteps: accepted=20, rejected=0, total=20
nodes: user=1, subckt=0, model=0, total=1
dctran density=100.0%, ac density=100.0%
".end "
